lxqt-config-appearance/colorLabel.cpp

```diff
@@ -25,13 +25,17 @@
 
 #include "colorLabel.h"
 #include <QColorDialog>
+#include <QStyleOptionFrame>
+#include <QPainter>
 
 ColorLabel::ColorLabel(QWidget* parent, Qt::WindowFlags f)
     : QLabel(parent, f)
 {
     setFrameStyle(QFrame::Panel | QFrame::Sunken);
+    setLineWidth(1);
     setFixedWidth(100);
```

Comment:
Do we still have fixed sizes?

Reply:
Is it okay with? https://github.com/lxqt/lxqt-config/issues/402
Scale factor seems to be no excuse to avoid default sizes

Reply:
What would a different implementation looks like ?

Reply:
@luis-pereira 
If you asked me, I didn't understand your question.

IMHO, avoiding stylesheets and respecting the widget style is a good practice. For that, I just did what Qt does in its color dialog.

As for the scale factor, it doesn't have any relevance here.

Reply:
@tsujan I was asking @shlyakpavel how he would implement the `setLineWith`. The stylessheet stuff is self-evident.

Reply:
Agree on stylesheet being wrong. However, would stick to the default border.

Reply:
@shlyakpavel 
The default border width *is* 1. Here, I was only explicit about it because I think it's good to be explicit.

Reply:
> @shlyakpavel
> I think it's good to be explicit.

I don't. This may vary between systems so we should better keep it to look native and integrated.

Reply:
@shlyakpavel 
Qt doc: https://doc.qt.io/qt-5/qframe.html#lineWidth-prop. Quoting: "The default value is 1." And it has nothing to do with integration or native look. *This PR* does.

A visual comparison that shows the border is identical to that of Qt's color dialog:

![border](https://user-images.githubusercontent.com/981076/88826241-bef61f80-d1dd-11ea-9e9c-24e27cae41ce.png)

Reply:
@tsujan ok, it's up to you